src/routes/Category/CategoryListing.tsx

```diff
@@ -10,6 +12,26 @@ const VISIBLE_ITEM_COUNT = 6;
 const CategoryListing = () => {
   const { data, isLoggedIn } = useLoaderData<LoaderData>();
   const [visibleItemCount, setVisibleItemCount] = React.useState(VISIBLE_ITEM_COUNT);
+  const categoryRoute = useCategoryRoute('CategoryListing');
+  const title = categoryRoute?.handle?.title;
+  const description = categoryRoute?.handle?.description;
+
+  const richTextClasses = tc([
```

Comment:
`richTextClasses` seems to be in several places, would it be possible to use only one version of this kind const in multiple places?